docs/planning/HB2B_StressStrain_peak0_Detailed.csv

```diff
@@ -2,6 +2,8 @@
 # Direction 11 1320,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 # Direction 22: 1321 1322,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 # Direction 33: 1329,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+# E: 285,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
right now we are spelling out the Young's modulus with units, just confirming that only E without units is the preferred choice